module/renderer/src/commonMain/kotlin/org/cru/godtools/shared/renderer/content/RenderLink.kt

```diff
@@ -0,0 +1,20 @@
+package org.cru.godtools.shared.renderer.content
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import org.cru.godtools.shared.tool.parser.model.Link
+
+@Composable
+fun ColumnScope.RenderLink(link: Link, modifier: Modifier = Modifier) {
+    RenderTextNode(
+        link.text,
+        modifier = modifier
+            .padding(horizontal = Horizontal_Padding)
+            .fillMaxWidth()
```

Comment:
per our Slack discussion you will need to set a minimum height for this to be `48.dp`. you can define a local constant for the height directly in this file using `private val Link_Minimum_Height = 48.dp`